internal/action/insert.go

```diff
@@ -89,20 +89,20 @@ func (s *Action) insert(ctx context.Context, c *cli.Context, name, key string, e
 }
 
 func (s *Action) insertStdin(ctx context.Context, name string, content []byte, appendTo bool) error {
-	var sec *secret.MIME
+	var sec gopass.Secret // *secret.MIME
 	if appendTo && s.Store.Exists(ctx, name) {
 		eSec, err := s.Store.Get(ctx, name)
 		if err != nil {
 			return ExitError(ExitDecrypt, err, "failed to decrypt existing secret: %s", err)
 		}
-		sec = eSec.MIME()
+		sec = eSec // .MIME()
 		sec.Write(content)
 	} else {
 		plain, err := secparse.Parse(content)
```

Comment:
I think we should stop parsing the input, or at least make it conditional.

Reply:
Otherwise I could handle the "stop parsing the input" in another PR to have a cleaner history?

Reply:
I tend to agree. But I'd first like to get everything to work before making that call.